users/templates/registration/login.html

```diff
@@ -25,8 +25,7 @@
   <div id="formContent">
 
     <div class="fadeIn first" id="login-header">
-      <img src="{% static 'users/images/logo-login.png' %}" alt="Logo icon" />
-      <img src="{% static 'users/images/name-login.png' %}" alt="Name brand Icon" />
+      <img src="{% static 'users/images/Logo-full-CP.png' %}" alt="Logo icon" />
```

Comment:
Please change name of this file to "full-cp-logo.png`

Reply:
Done